src/openscm_runner/adapters/fair2/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
+Module supporting the FAIR adapter
```

Comment:
```suggestion
Module supporting the FAIR2 adapter
```

?? Or does the distinction not matter because you can't install FaIR1.X and FaIR2 simultaneously anyway?